src/EnergyPlus/HVACDXSystem.cc

```diff
@@ -2604,7 +2595,7 @@ namespace HVACDXSystem {
         DXCoolingSystem(DXSystemNum).CycRatio = CycRatio;
         DXCoolingSystem(DXSystemNum).DehumidificationMode = DehumidMode;
         DXCoolingSystem(DXSystemNum).SpeedNum = SpeedNum;
-    }
+    } // namespace HVACDXSystem
```

Comment:
For some reason, this comment was added here. Not sure why but I have seen this elsewhere in code I was working on and have to delete it since it's obviously in the wrong place.